examples/platform/silabs/efr32/BaseApplication.cpp

```diff
@@ -299,6 +299,87 @@ void BaseApplication::FunctionEventHandler(AppEvent * aEvent)
     }
 }
 
+bool BaseApplication::ActivateStatusLedPatterns()
+{
+    bool isPatternSet = false;
+#if defined(ENABLE_WSTK_LEDS) && defined(SL_CATALOG_SIMPLE_LED_LED1_PRESENT)
```

Comment:
This define should be removed. You can also leave it you want but it will be remove upon merging the BaseApplication.

Reply:
Currently, LEDWidget sStatusLED; is encapsulated in the same ifdef and so are the includes for the LedWidget and `sl_simple_led_instances`
I'll leave it here. When it isn't needed anymore we can quickly Find and delete all the ENABLE_WSTK_LEDS ifdefs